app/options.html

```diff
@@ -18,43 +18,43 @@
                 <div class="col-md-8 col-md-offset-2">
                     <div class="jumbotron">
                         <img src="images/icon-128.png" class="pull-right" id="icon128"/>
-                        <h1>FullyFeedly</h1>
-                        <p>Show the full content of the articles in Feedly.com</p>
+                        <h1 data-message="appName">FullyFeedly</h1>
+                        <p data-message="appDescription">Show the full content of the articles in Feedly.com</p>
```

Comment:
I assume that we don't need to add an English translation for these since they are already in that language. Right?

I wonder if we could do the same for the buttons "Show article preview" in the page.